core-services/core_services.tf

```diff
@@ -19,6 +19,12 @@ data "aws_route53_zone" "hosted_zones" {
   zone_id  = each.key
 }
 
+resource "kubernetes_namespace_v1" "md-core-services" {
```

Comment:
This is the only way to ensure this namespace gets created now since everything else is conditional (previously it was getting made by the first helm chart to run and need the namespace).

We also now are guaranteed to need the namespace since the `massdriver` service_acount is getting deployed there instead of the `default` namespace.

This will require a simple `terraform import` on clusters

Reply:
In my experience, this butts heads pretty bad with the helm modules in core-services. They _can_ create the namespace first and if that happens, this resource fails. 

I went down this path on the gcp side and ran into that issue. ^
https://github.com/massdriver-cloud/terraform-modules/pull/86/files

A keen eye would notice that there's no namespace there. :) That's because I tried a few `depends_on` and all of them felt gross. I then realized that we don't currently set the namespace (like you mention), and that I was just trying to fix a VSCode linter warning. Specifically for the service account we create. Anyway, something I ran into provisioning this a few times and a possible race condition.

Also, note to self and orc team. The code for the k8s service account could probably be a k8s-* module and not live in each cloud codebase. We can make a ticket for later if we want to do that.

Reply:
If you reference the attribute `kubernetes_namespace_v1.md-core-services.metadata.0.name` for the `namespace` in the core-services helm charts (instead of a local string) it should be an implicit `depends_on`.  This should prevent collisions on new clusters (though there will obviously be collisions on existing clusters, hence the need for terraform import when this rolls out).

Reply:
Doh... That's great, I'll give this a shot locally.